tests/3d_examples/test_3d_passive_cantilever_neohookean/CMakeLists.txt

```diff
@@ -29,46 +29,42 @@ file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/regression_test_tool/CantileverObserver_Po
         DESTINATION ${BUILD_INPUT_PATH})
 
 aux_source_directory(. DIR_SRCS)
-ADD_EXECUTABLE(${PROJECT_NAME} ${DIR_SRCS})
 
-add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME}
-                WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+if(EMSCRIPTEN)
+	ADD_EXECUTABLE(${PROJECT_NAME} ${DIR_SRCS} ${TBB_OBJECTS})
+	target_link_libraries(${PROJECT_NAME} sphinxsys_static_3d)
+else()
+	ADD_EXECUTABLE(${PROJECT_NAME} ${DIR_SRCS})
+	add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME})
 
-if(NOT SPH_ONLY_STATIC_BUILD) # usual dynamic build
-	if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-                                set_target_properties(${PROJECT_NAME} PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY "${EXECUTABLE_OUTPUT_PATH}")
-		target_link_libraries(${PROJECT_NAME} sphinxsys_3d)
-		add_dependencies(${PROJECT_NAME} sphinxsys_3d)
-	else(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-		if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-			target_link_libraries(${PROJECT_NAME} sphinxsys_3d stdc++)
-		else(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-			target_link_libraries(${PROJECT_NAME} sphinxsys_3d stdc++ stdc++fs dl)
-		endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+	if(NOT STATIC_BUILD) # usual dynamic build
```

Comment:
Shouldn't it be SPH_ONLY_STATIC_BUILD here ? or is it specific to wasm compilation

Reply:
true